src/lib.rs

```diff
@@ -45,6 +45,16 @@ pub const MEMO_SIZE: usize = 512;
 /// The size of the authentication tag used for note encryption.
 pub const AEAD_TAG_SIZE: usize = 16;
 
+/// The size of a compact note.
+pub const COMPACT_NOTE_SIZE: usize = 1 + // version
+    11 + // diversifier
+    8  + // value
+    32; // rseed (or rcm prior to ZIP 212)
+/// The size of NotePlaintextBytes.
+pub const NOTE_PLAINTEXT_SIZE: usize = COMPACT_NOTE_SIZE + 512;
+/// The size of an encrypted note plaintext.
+pub const ENC_CIPHERTEXT_SIZE: usize = NOTE_PLAINTEXT_SIZE + AEAD_TAG_SIZE;
+
```

Comment:
So, these values need to be introduced on the implementation side (sapling), not in the interface side.

Reply:
after all, the idea is to support multiple note sizes.